closed/autoconf/version-numbers

```diff
@@ -0,0 +1,28 @@
+# ===========================================================================
+# (c) Copyright IBM Corp. 2020, 2020 All Rights Reserved
+# ===========================================================================
+# 
```

Comment:
Please remove the trailing whitespace (or the whole line); same goes for line 20.

Reply:
Fixed